woocommerce/google-listings-and-ads
# Re-connecting WPCOM Account - Adjust the linked accounts section in settings

### Changes proposed in this Pull Request:

It's a part of frontend in #775

- Move `<WPComAccountCard>`, `<GoogleMCAccountCard>`, and `<GoogleAdsAccountCard>` to the shared component folder.
   - 💡  Moved a lot of files, adjusted several files, and internalized WPCOM appearance to `<AccountCard>`. This is to give a little better consistency to their composition of components across all similar account cards.
- Change the linked accounts section in settings to use Card UI to present each account.

### Screenshots:

#### 📷  Not yet connected to a Google Ads account
![image](https://user-images.githubusercontent.com/17420811/160577915-f061b755-40de-4d06-8d93-211a9005c1e9.png)

#### 📷  Has a connected Google Ads account
![image](https://user-images.githubusercontent.com/17420811/160577652-1f0477de-ebde-4b33-9a28-c9b1b9473951.png)

### Detailed test instructions:

1. Go to the Settings page.
2. Check if the UI of **Linked Accounts** section is the same as the design in Figma.
3. Go to the onboarding setup and Google ads account setup pages.
4. Check if all account cards are displayed as the same as before.

### Additional details:

Figma link: eQ9O4m2flzrcAiDMXkOW0m-fi-6257%3A116720

### Changelog entry

> Tweak - Adjust the UI of the linked accounts section on the Settings page.
